app/api/chat/route.ts

```
@@ -34,6 +35,17 @@ export async function POST(req: NextRequest) {
         aiModel = openrouter(modelConfig.modelId);
         break;
 
+      case 'litellm':
+        const baseURL =
+          headersList.get('X-LiteLLM-Base-Url') ??
+          process.env.LITELLM_BASE_URL;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)